apps/cowswap-frontend/src/entities/bridgeProvider/bridgeProvidersAtom.ts

```diff
@@ -0,0 +1,7 @@
+import { atom } from 'jotai'
+
+import { isProd } from '@cowprotocol/common-utils'
+
+import { bridgeProviders } from 'tradingSdk/bridgingSdk'
+
+export const bridgeProvidersAtom = atom(isProd ? [bridgeProviders[0]] : bridgeProviders)
```

Comment:
Lol, that's your feature flag?

I'm joking, a ff might be overkill. This should be fine for now, until we have a new provider for real.
Although it could make testing harder until then.

Reply:
🙈 
I didn't want to add an Updater for that.
We agreed with Elena that this is ok, she only will need to test in on Staging.